src/modules/launcher/PowerLauncher/Helper/NativeEventWaiter.cs

```diff
@@ -19,6 +21,9 @@ public static void WaitForEventLoop(string eventName, Action callback)
                 {
                     if (eventHandle.WaitOne())
                     {
+                        Log.Info("Successfully waited for POWER_LAUNCHER_SHARED_EVENT", MethodBase.GetCurrentMethod().DeclaringType);
+
+                        // Log successfull wait
```

Comment:
Please fix misspelling successfull -> successfully

Reply:
IMHO the comment is unnecessary, as it doesn't really give any extra info